src/utils/session.ts

```diff
@@ -0,0 +1,61 @@
+import { CONNECTION_TYPES, IS_EXTENSION, IS_OFFSCREEN_TAB } from '@/constants';
+import { getSessionEncryptionKey as getSessionEncryptionKeyOffscreen } from '@/offscreen/popupHandler';
+import { exportEncryptionKey, importEncryptionKey } from './crypto';
+
+const SESSION_STORAGE_KEYS = {
+  exportedEncryptionKey: 'exportedEncryptionKey',
+  sessionExpires: 'sessionExpires',
+};
+
+const storageSession = (browser.storage as any)?.session;
```

Comment:
Please bare in mind that after recent change from @CedrikNikita we are not using browser storage anymore.

Reply:
This is a separate storage type. The fact that we are not using browser storage shouldn't affect the `browser.storage.session` storage type